src/content/about/metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Metrics
+weight: 4
+---
+
+- [Per Worker Script Metrics](#per-worker-script-metrics)
+  - [Invocation statuses](#invocation-statuses)
+  - [Worker requests](#worker-requests)
+  - [CPU time](#cpu-time)
+- [Per Zone Metrics](#per-zone-metrics)
+  - [Total requests](#total-requests)
+  - [Status codes](#status-codes)
+  - [Bandwidth](#bandwidth)
+  - [Subrequests](#subrequests)
+- [GraphQL](#graphql)
+
+Workers metrics show performance and usage of your Workers and help diagnose issues.
+
+Workers Metrics are available in Per Zone and Per Worker script contexts.
+
+## Per Worker Script Metrics
+
+Aggregates request data for an individual Worker script across all zones including your workers.dev subdomain. On your [Workers dashboard](https://dash.cloudflare.com/?account=workers/overview), click on any Worker to view its metrics.
+
+Per Worker script metrics can be inspected by time ranges within the last 30 days. The dashboard includes the charts and information described below.
+
+### Invocation statuses
```

Comment:
Should organize in the same order and titles as the UI

Reply:
After reading through the doc, I'm afraid that reorganizing it this way confuse things because the invocation statuses section provides necessary context for the rest of the document.  In other words, if I'm reading the page like a book, and sections are ordered based on the UI, each section will need to refer to the `Invocation statuses` for clarification.  

Immediately introducing and disambiguating invocation statuses takes care of this and decouples the docs section ordering from future UI updates that might re-arrange things visually.

Reply:
We don't expect this to be read like a book since it's not a tutorial. It serves as reference. The first section in the UI is Requests, so I suggesting that one go first. The order isn't as big of deal as the titles actually aligning with what's in the UI.

Reply:
This would mean when UI changes, the ordering of sections in the documentation has to change.  Is this a pattern we want to commit to?

It seems fragile and somewhat arbitrary. Readers won't know what an invocation status is without reading the end of the document because of the order the charts appear in the UI.